docs/tutorials.rst

```diff
@@ -250,41 +258,39 @@ your application.
 .. code-block:: python
    :linenos:
 
-    import random
-    import string
-
-    import cherrypy
-
-    class StringGenerator(object):
+   import random
```

Comment:
Why prefer three spaces to four for this indentation? The need for this change is not obvious to me.